### ISSUE TITLE
Some sites don't load properly

### ISSUE THREAD

**Issue by stardast** [samuong/alpaca#116]
I'm using alpaca_v2.0.2_linux-amd64 and websites like figma.com and strigo.io aren't loading.

**Comment by samuong** [comment]
I've been able to use figma through alpaca in the past. Are you able to provide more info about what your doing? The following information would be useful:

- what does your pac file look like?
- what do you see in the alpaca logs when you visit these sites?
- what errors do you see in your browser?

**Comment by stardast** [comment]
To make sure it's not a proxy problem I tried with this pac on my home connection:
`function FindProxyForURL(url, host) { return "DIRECT"; }`
alpaca log attached:
[logFigma.txt](https://github.com/samuong/alpaca/files/12617129/logFigma.txt)
the page remains partially loaded.

Thanks

**Comment by samuong** [comment]
From your logs, I can see you get a few errors saying `write: broken pipe`. It's a bit hard to say exactly what's causing this, usually when I've seen it, it's a connectivity issue. But for you, it seems like this is mostly just happening with sites like doubleclick, which I don't think would affect your Figma page from loading (but you also get one error on `s3-alpha.figma.com`)

Figma seems to work ok for me, at least if I just visit the front page. Are you doing more than that, maybe logging in?

![Screenshot from 2023-09-15 19-31-16](https://github.com/samuong/alpaca/assets/8053866/5e33da28-8240-41e5-a27a-cf4b2493bd48)

**Comment by stardast** [comment]
Yes, the half-loaded page and the one after login. Can I enable a more detailed logging level to give you more information?

**Comment by samuong** [comment]
There's not really any way to enable more logging in alpaca itself, but it might be useful if you could share a screenshot of the "half-loaded page", as well as a screenshot of the DevTools Network tab showing which requests failed.
